src/components/button/ButtonDisplay.js

```diff
@@ -0,0 +1,28 @@
+import Button from './Button';
+
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons';
```

Comment:
check faCoffee can be imported as,
import { faCoffee } from '@fortawesome/free-solid-svg-icons/faCoffee';

The way you have done, while webpack will make build of application, it can add whole font-awesome lib.

Read more about webpack tree shaking.